Makefile

```diff
@@ -128,6 +128,57 @@ clean:
 		$(CURDIR)/bin \
 		$(CURDIR)/pkg
 
+# Run controller tests
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+ctrl-test: ctrl-generate ctrl-manifests
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/hack/setup-envtest.sh
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
```

Comment:
curious if this leaves cover.out hanging out somewhere?

Reply:
good catch. will drop it!! it definitely is a dangling file left behind